packages/mask/background/services/identity/persona/sign.ts

```diff
@@ -19,6 +19,7 @@ export async function signWithPersona<T>(
     type: SignType,
     message: T,
     identifier?: ECKeyIdentifier,
+    source?: string,
```

Comment:
```suggestion
```

Reply:
No, This parameter is required for the signature page.